packages/core/testing/src/component_fixture.ts

```diff
@@ -194,7 +194,7 @@ export class ComponentFixture<T> {
    */
   getDeferBlocks(): Promise<DeferBlockFixture[]> {
     const deferBlocks: DeferBlockDetails[] = [];
-    const lView = (this.componentRef.hostView as any)['_lView'];
+    const lView = (this.componentRef.hostView as ViewRef<unknown>)._lView;
     getDeferBlocks(lView, deferBlocks);
```

Comment:
As an option we can avoid accessing `LView` and instead pass the `ViewRef` to the `getDeferBlocks`. WDYT?
```suggestion
    getDeferBlocks(this.componentRef.hostView, deferBlocks);
```